src/main/scala/wolfendale/nunjucks/filters/package.scala

```diff
@@ -56,13 +56,37 @@ package object filters {
     }
   }
 
+  val default: Filter = Filter { (value: Value, args) =>
+    if (!Seq(1,2).contains(args.parameters.size)) {
+      throw new RuntimeException(s"Filter default requires either one argument (default) or two arguments (default, falsy)")
```

Comment:
I can't find a case which actually results in an error being thrown from the filter.